CHANGELOG.md

```diff
@@ -5,27 +5,36 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Added support for PureScript 0.14 and dropped support for all previous versions (#66)
 
 New features:
+- Added `buildFromScratch` for building from an empty record (#53)
+- Added `new` and `run` to `Record.ST` (#71)
+- Added `flip` function (#73)
 
 Bugfixes:
 
 Other improvements:
+- Updated `Record.Builder.merge` and `Record.Builder.union` to override existing fields (#73)
```

Comment:
I think this change is breaking? @kl0tl could you clarify?

Reply:
This changes the behaviour of Record.Builder.merge and Record.Builder.union, so that

```purescript
build (merge { x: 1, y: "y" }) { y: 2, z: true }
```

has now type `{ x :: Int, y :: String, z :: Boolean }` instead of `{ x :: Int, y :: Int, z :: Boolean }` and

```purescript
build (union { x: 1, y: "y" }) { y: 2, z: true }
```
has now type `{ x :: Int, y :: String, y :: Int, z :: Boolean }` instead of `{ x :: Int, y :: Int, y :: String, z :: Boolean }`.

This is indeed a breaking change.

Reply:
I edited Cyril's comment above to make the differences easier to see.

@kl0tl What's a good way to summarize this change?

Reply:
What do you think of:

> Updated `Record.Builder.merge` and `Record.Builder.union` so that they behave like `Record.merge` and `Record.union`: fields from the argument override those of the record being built in case of overlaps.

Reply:
Sounds good to me.